bazel/internal.bzl

```diff
@@ -0,0 +1,14 @@
+"""
+This module contains internal helpers that should not be used outside of the
+scripts in the `bazel/` directory.
+"""
+
+def redpanda_copts():
```

Comment:
Can we add the existing copts too:

https://github.com/redpanda-data/redpanda/blob/6094a4eaefd7b057747e1d463a4bcf3bb88207c2/cmake/v_library.cmake#L5

Reply:
Added in https://github.com/dotnwat/redpanda/pull/20. Look good?

Reply:
Yeah LGTM